src/index.ts

```diff
@@ -157,13 +222,63 @@ export class CheckoutInstance {
     }
   }
 
+  private handleClick() {
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+    // @ts-ignore
+    this.dialog.showModal()
+  }
+  /**
+   * Creates popup modal in the DOM to host the Checkout iframe.
+   *
+   * If the Checkout becomes completed or if the user clicks cancelled, automatically closes the popup.
```

Comment:
The "becomes completed" should be removed, right?
Also I think "cancelled" should be "cancel", or some other name for the close popup button.

Reply:
yes. done